packages/framework/src/Migrations/Version20220725104726.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20220725104726 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('ALTER TABLE delivery_addresses ALTER first_name SET NOT NULL');
```

Comment:
We should handle existing values for all fields. 
I suggest to set an empty string like this 

```
$this->sql('UPDATE delivery_addresses SET first_name = \'\' WHERE first_name IS NULL');
$this->sql('ALTER TABLE delivery_addresses ALTER first_name SET NOT NULL');
```

Reply:
I edit migration by your suggestion but I am not sure about `country_id`. I added random id but i am not sure if it's correct solution. Check it again please https://github.com/shopsys/shopsys/pull/2494/commits/45ffcba392b9f930e4e53af6d923e954d535ae9c

Reply:
🤔  hmm that's quite a brain-teaser.

What about something like 

```
UPDATE delivery_address
SET country_id = (
    SELECT cd.country_id
    FROM customers c
    JOIN country_domains cd ON c.domain_id = cd.domain_id
    WHERE c.id = delivery_address.customer_id
      AND cd.enabled = TRUE
    ORDER BY cd.country_id ASC
    LIMIT 1
)
WHERE country_id IS NULL;
```

it's still arbitrary country, but at least country that's enabled on the customer's domain 🤷 (I did not test the migration, so there may be some mistake)

What do you think?

Reply:
Selecting from countries only that are allowed in domain is good idea 👌 It's still not 100% solution but I think it is good now. Changed in [commit](https://github.com/shopsys/shopsys/pull/2494/commits/98d55d13c19744e6753a27a6f160b82ddc47d500) and tested